static/swaggerui/swagger.json

```diff
@@ -0,0 +1,456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is the first version of Kube-green API.",
+    "version": "1.0.0",
+    "title": "Kube-green API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "email": "apiteam@swagger.io"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "",
+  "basePath": "/v1",
+  "tags": [
+    {
+      "name": "kube-green",
+      "description": "Make your k8s clusters more green",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "https://mia-platform.eu/"
+      }
+    }
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "paths": {
+    "/cpu_usage": {
+        "summary": "CPU usage",
+        "description": "Get CPU usage",
+        "operationId": "getCpuUsage",
+        "produces": [
+          "application/xml",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "namespace",
+            "in": "query",
+            "description": "name of namespace",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "start",
+            "in": "query",
+            "description": "Start timestamp (unix timestamp in milliseconds)",
+            "required": true,
+            "type": "integer",
+            "format": "int64"
+          },
+          {
+            "name": "end",
+            "in": "query",
+            "description": "End timestamp (unix timestamp in milliseconds)",
+            "required": true,
+            "type": "integer",
+            "format": "int64"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "type": "array"
+            }
+          },
+          "400": {
+            "description": "Invalid status value"
+          }
+        }
+      },
+   
+    "/all_active_pods": {
+        "summary": "All Active Pods",
+        "description": "Get All Active Pods",
+        "operationId": "getAllActivePods",
+        "produces": [
+          "application/xml",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "namespace",
+            "in": "query",
```

Comment:
There is too much duplication of code around, I might actually think about code generation if there is no way to compensate for this, it makes me saad :(

Reply:
I couldn't find anything to remove the code dublicator on json file -_-

Reply:
:((, why god why? :(